mep/people/templates/people/membership_activities.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static %}
+
+{% block page-subtitle %}{{ member.name }} · Library Members · {% endblock %}
+
+{% block js %}
+<script type="application/ld+json">{{ page_jsonld|safe }}</script>
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.name style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+<aside>
+  {% include 'snippets/breadcrumbs.html' %}
+</aside>
+<nav class="tabs" aria-label="sections">
+    <li class="tab">
+        <a href="{% url 'people:member-detail' pk=member.pk %}">biography</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'people:membership-activities' pk=member.pk %}">activities</a>
+    </li>
+    <li class="tab">
+        <a href="#">cards</a>
+    </li>
+</nav>
+
+<nav class="tabs subsection" aria-label="Activities">
```

Comment:
probably need to look into whether there is a way to programmatically associate this with the main tab nav (with aria) so that the screen reader user is aware that it is a sub-nav of the main tabs. `aria-owns`, maybe? hopefully we can find a similar pattern. the tricky thing is that visually they're tabs, but for all other purposes (including SRs) they're a nav/tree of links.